kubernetes/Dockerfile

```diff
@@ -0,0 +1,63 @@
+FROM centos:centos7
+
+ENV CDH_VERSION     5.7.6
+ENV JAVA_VERSION    1.8.0
```

Comment:
A Dockerfile has been committed to Kylin. Could yout reuse it ?

Reply:
Emm... the committed dockerfile is self-contained and work with Apache version instead of CDH version. It seems to prepare for experiment, I think.

Reply:
Hi, @TempleZhou , what do you hope? We can work to make it usable in production Environment.

Reply:
Hi @TempleZhou, how about split docker image as 2 part: one is base image, and the other is kylin image.
Kylin docker image base on base image (hadoop-client image) by 'FROM'.
And we can also provide 2 base image: one is apache version, the other is CDH version.
It will benefit users who owns different version of hadoop version for their environment.
What do you think?